src/renderer/renderElement.ts

```diff
@@ -989,7 +989,28 @@ export const renderElement = (
           element,
           renderConfig,
         );
+
+        if (
+          // angle is 0 -> always disable smoothing
+          !element.angle ||
+          // or check if angle is a right angle in which case we can still
+          // disable smoothing without adversely affecting the result
+          isRightAngle(element.angle)
+        ) {
+          // Disabling smoothing makes output much sharper, especially for
+          // text. Unless for non-right angles, where the aliasing is really
+          // terrible on Chromium.
+          //
+          // Note that `context.imageSmoothingQuality="high"` has almost
+          // zero effect.
+          //
+          context.imageSmoothingEnabled = false;
+        }
+
         drawElementFromCanvas(elementWithCanvas, rc, context, renderConfig);
+
+        // reset
+        context.imageSmoothingEnabled = true;
```

Comment:
```suggestion
        context.imageSmoothingEnabled = currentImageSmoothingStatus;
```
I would just add this, in case the default value is changed in the future.